Readme.md

```diff
@@ -2,116 +2,188 @@
 
 [![npm](https://img.shields.io/npm/v/richtypo.svg)](https://www.npmjs.com/package/richtypo) [![Build Status](https://travis-ci.org/sapegin/richtypo.js.svg)](https://travis-ci.org/sapegin/richtypo.js)
 
-Richtypo adds non-breaking spaces in the right places, `<nobr>` and `<abbr>` tags and wraps ampersands with a CSS class for special styling. It expects that your texts already have the right quotes, em-dashes and other symbols (you can use [Typography Keyboard Layout](http://ilyabirman.net/projects/typography-layout/)).
+Richtypo is a framework for adding rules to plain or HTML text. You can define your own set of rules or import one of the [rules packages](##rule-packages).
+
+Richtypo was made with typographic rules in mind to render HTML text following the same guidelines as a book being printed (eg adding a non breaking space to avoid orphans), but rules are actually just functions that can transform text in any desired way.
 
 ## Features
 
-- Rules for English and Russian languages
-- Non-breaking spaces after prepositions and conjunctions, before em-dash, etc.
-- `<nobr>` for words with hyphens
-- CSS classes for ampersands
-- `<abbr>` for abbreviations
+- Works with plain or HTML text
 - Takes care of your HTML tags
-- Simple typographer (quotes, em-dash, etc.) for user generated content (like comments)
+- Extensible through third-party rule packages
+- Works server-side
 - No dependencies
 
-## Example
+## Usage
 
 ```javascript
-const richtypo = require('richtypo');
-const beautiful = richtypo.rich(
-  'Welcome to the world of beautiful web typography — only with Richtypo.'
-);
-const awesome = richtypo.title(
-  'Beautiful &amp; Awesome Web Typography with “Richtypo”'
-);
-const ok = richtypo.lite('"Richtypo" - awesome!');
+import richtypo from 'richtypo';
+import rules from 'richtypo-rules-en';
+
+const text = 'There are 1000 "ways" to extend Richtypo.';
+
+// the all rule is defined in richtypo-rules-en
+// and executes all the rules from the package.
```

Comment:
executes → contains? `richtypo` function executes them ;-)

Reply:
I changed the Readme example text (if you have a better suggestion let me know!)
f6700a1f0b352135f7681cb48600f369a481216b